packages/block-library/src/template-part/variations.js

```diff
@@ -38,9 +24,33 @@ const variations = [
 			"The footer template defines a page area that typically contains site credits, social links, or any other combination of blocks. Since it's a global element it can be present across all pages and posts."
 		),
 		icon: footer,
-		isActive: createIsActiveBasedOnArea( 'footer' ),
-		scope: [],
+		attributes: { area: 'footer' },
+		scope: [ 'inserter' ],
 	},
 ];
 
+/**
+ * Add `isActive` function to all `Template Part` variations, if not defined.
+ * `isActive` function is used to find a variation match from a created
+ *  Block by providing its attributes.
+ */
+variations.forEach( ( variation ) => {
+	if ( variation.isActive ) return;
+	variation.isActive = ( blockAttributes, variationAttributes ) => {
```

Comment:
I know this predates the PR, but the `isActive` nomenclature is quite confusing to me. What does it represent?

Reply:
From the docs:
>A function that accepts a block’s attributes and the variation’s attributes and determines if a variation is active. This function doesn’t try to find a match dynamically based on all block’s attributes, as in many cases some attributes are irrelevant. An example would be for embed block where we only care about providerNameSlug attribute’s value.

This is useful for cases you want to use information from the block variation, **after a block’s creation**. In this case though we have a combination with information from variation and entity attributes (`area`).